### PR TITLE
Add Description column to Tasks Panel

### PR DESCRIPTION
**Pull request by smbende1**

### CODE REVIEW

File: src/net/sf/memoranda/ui/TaskTableModel.java [hunk 1]
```diff
@@ -86,19 +87,21 @@ public Object getValueAt(Object node, int column) {
         case 1:
             return t;
         case 2:
-            return t.getStartDate().getDate();
+        	return t.getDescription();
```
Comment (dtapia4) [inline src/net/sf/memoranda/ui/TaskTableModel.java]: Is it just me or is the indentation a little off?

File: src/net/sf/memoranda/ui/TaskTableModel.java [hunk 2]
```diff
@@ -86,19 +87,21 @@ public Object getValueAt(Object node, int column) {
...
-        case 5:
-            return getPriorityString(t.getPriority()); 
         case 6:
+            return getPriorityString(t.getPriority()); 
+        case 7:
             return getStatusString(t.getStatus(CurrentDate.get()));
-        case 7:            
+        case 8	:            
             //return new Integer(t.getProgress());
 			return t;
```
Comment (dtapia4) [inline src/net/sf/memoranda/ui/TaskTableModel.java]: fix indentation

**Comment by dtapia4** [comment]
Sean, I made a couple of comments, very minor. Everything else looks good though.

**Comment by smbende1** [comment]
Fixed the indentation issues identified and also corrected some stylistic problems with if/else statement blocking.

**Comment by dtapia4** [comment]
Looks good.

**Pull request merged**